CrewChiefV4/UserInterface/PropertiesForm.Designer.cs

```diff
@@ -50,43 +60,130 @@ private void InitializeComponent()
             this.flowLayoutPanel1.AutoScroll = true;
             this.flowLayoutPanel1.AutoSize = true;
             this.flowLayoutPanel1.AutoSizeMode = System.Windows.Forms.AutoSizeMode.GrowAndShrink;
-            this.flowLayoutPanel1.Dock = System.Windows.Forms.DockStyle.Top;
-            this.flowLayoutPanel1.Location = new System.Drawing.Point(0, 0);
+            this.flowLayoutPanel1.Dock = System.Windows.Forms.DockStyle.Fill;
+            this.flowLayoutPanel1.Location = new System.Drawing.Point(3, 31);
             this.flowLayoutPanel1.Name = "flowLayoutPanel1";
-            this.flowLayoutPanel1.Size = new System.Drawing.Size(975, 0);
+            this.flowLayoutPanel1.Size = new System.Drawing.Size(969, 612);
             this.flowLayoutPanel1.TabIndex = 9;
             // 
-            // panel1
+            // textBox1
             // 
-            this.panel1.Controls.Add(this.button1);
-            this.panel1.Dock = System.Windows.Forms.DockStyle.Bottom;
-            this.panel1.Location = new System.Drawing.Point(0, 604);
-            this.panel1.Name = "panel1";
-            this.panel1.Size = new System.Drawing.Size(975, 48);
-            this.panel1.TabIndex = 10;
+            this.textBox1.Location = new System.Drawing.Point(780, 3);
+            this.textBox1.Margin = new System.Windows.Forms.Padding(0, 3, 3, 3);
+            this.textBox1.Name = "textBox1";
+            this.textBox1.Size = new System.Drawing.Size(189, 20);
+            this.textBox1.TabIndex = 11;
+            this.searchBoxTooltip.SetToolTip(this.textBox1, resources.GetString("textBox1.ToolTip"));
```

Comment:
should this tool tip be retrieved from the ui_strings with something like 
Configuration.getUIString("search_box_tool_tip");